eng/helix/content/runtests.cmd

```diff
@@ -26,8 +24,8 @@ set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 set DOTNET_MULTILEVEL_LOOKUP=0
 set DOTNET_CLI_HOME=%HELIX_CORRELATION_PAYLOAD%\home
 
-set PATH=%DOTNET_ROOT%;!PATH!;%HELIX_CORRELATION_PAYLOAD%\node\bin
-echo Set path to: %PATH%
```

Comment:
Have you confirmed the path issues no longer occur?  There's a few queues that have spaces due to jdk paths which used to die miserably, issues might only show up if you run the full matrix

Reply:
1. This `SET` set command didn't cause that problem; we invoke `dotnet-install.ps1` w/ `-noPath` to avoid that
2. The issue here apparently was parentheses in the `%PATH%` value causing script parsing problems; I chose the more common way to deal with that.

FYI&hellip;
``` text
Set path to: "C:\h\w\AE0A0984\p\sdk\x64;C:\python3.7.0\lib\site-packages\pywin32_system32;C:\python3.7.0\Scripts\;C:\python3.7.0\;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Users\runner\AppData\Local\Microsoft\WindowsApps;C:\h\w\AE0A0984\p\node\bin"
```

Reply:
BTW spaces are fine when using this quoted syntax too

Reply:
If you say so, that's how things started, but we'll find out soon enough :)

Reply:
But yeah I guess the issue I recall is with parens causing issues with IF parsing, not spaces so its only machines that have the `"C:\Program Files (x86)";`

Reply:
I'd suggest running this through the full helix queue before merging regardless when mucking with runtests.cmd

Reply:
We might be okay since we don't use the path inside an IF anymore in this script which I think we used to do

Reply:
This syntax works fine for all the cases you want to throw at it other than nesting in `if`s but I kicked off https://dev.azure.com/dnceng/public/_build/results?buildId=804251

Side note: That matrix only runs for the master branch and is completely on the floor. Hope #25650 gets it going again.

Reply:
RE: side note, is that a good idea to not run the matrix something like once a day in release branches, there's both risk in product platform regressions and even test only issues that cause the matrix to break will be hard to isolate when merging back to master if it never is run

Reply:
I agree w/ you @HaoK Something worth adding though not in this PR.